package.json

```diff
@@ -10,8 +10,10 @@
   "scripts": {
     "build": "webpack -d --config webpack.config.js --watch",
     "start": "nodemon server/index.js",
-    "seed": "node db/seedDB.js",
-    "test": "jest"
+    "seed": "psql postgres < schema.sql",
```

Comment:
do you have a data gen script? which command creates the data? just wondering since you would have to allocate the correct amount of space for node in order to generate and seed that much data